public/data/mukulojha.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Mukul Ojha",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/mukul314.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/mukul314",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Mukulsh69645170",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started coding",
+      "date": "December 2020",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started coding and learning.",
+      "url": "https://www.eddiejaoude.io/mukul314"
```

Comment:
The **URL you entered doesn't exist!**. I recommend placing your own URL here, or remove this URL section. (line no. `25`)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable